docker-compose.yml

```
@@ -0,0 +1,23 @@
+version: "2.4"
+
+services:
+  phoxicontrol:
+    image: photoneo/phoxicontrol
+    runtime: nvidia
```

Btw. You don't have to use Nvidia. So you can use your integrated graphics card
```suggestion
    devices:
      - /dev/dri:/dev/dri
```

---

In case you decide to use an integrated card the section on how to install Nvidia runtime can be deleted.

---

Do we still need opengl base image?